src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4509,6 +4509,8 @@ public partial class FileLoadException : System.IO.IOException
         public FileLoadException(string message, string fileName) { }
         public FileLoadException(string message, string fileName, System.Exception inner) { }
         protected FileLoadException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) : base(info, context) { }
+        [System.Security.SecurityCriticalAttribute]
```

Comment:
I've seen them in some reference assemblies so that's why I left them, but I believe you are right and they aren't looked by the compiler, so I'll just remove them then.